File: Data Structures & Algorithms/find-peak-element/submission-1.java
class Solution {
    public int findPeakElement(int[] arr) {
        int n = arr.length;
        for(int i=1;i<n-1;i++){
            if(arr[i]>arr[i-1] && arr[i]>arr[i+1]){
                return i;
            }
        }

        return arr.length-1;
    }
}

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-1.py
class Solution:
    def replaceElements(self, arr: List[int]) -> List[int]:
        n = len(arr)
        l2 = n*[0]
        max_right = -1
        for i in range(n-1,-1,-1):
            l2[i] = max_right
            max_right = max(max_right,arr[i])
        return l2